<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeviceDialog</class>
 <widget class="QDialog" name="AddDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>214</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>214</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>214</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add new device</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="lineEditUsername">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>60</y>
     <width>252</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>115</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>182</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddress">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>34</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Hostname / IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVersion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>138</y>
     <width>252</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Unknown</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>37</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>35</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>63</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescription">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>112</y>
     <width>252</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelVersion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>139</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>34</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="placeholderText">
    <string>5250</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>87</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>86</y>
     <width>252</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShadow">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>162</y>
     <width>248</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Shadow system</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDeviceName">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>8</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDeviceName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>11</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDeviceName</tabstop>
  <tabstop>lineEditAddress</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditUsername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>lineEditVersion</tabstop>
  <tabstop>checkBoxShadow</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>AddDeviceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDeviceName</sender>
   <signal>editingFinished()</signal>
   <receiver>AddDeviceDialog</receiver>
   <slot>lookupName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditAddress</sender>
   <signal>editingFinished()</signal>
   <receiver>AddDeviceDialog</receiver>
   <slot>lookupAddress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lookupAddress()</slot>
  <slot>lookupName()</slot>
 </slots>
</ui>
